params/config.go

```diff
@@ -1090,7 +1105,7 @@ type Rules struct {
 	IsVerkle                                                bool
 	IsOptimismBedrock, IsOptimismRegolith                   bool
 	IsOptimismCanyon, IsOptimismFjord                       bool
-	IsOptimismGranite                                       bool
+	IsOptimismGranite, IsOptimismHolocene                   bool
```

Comment:
nit: why two to a line?

Reply:
simply because it was done that way for prior forks 😬